cdc/sink/codec/canal/canal_json_encoder.go

```diff
@@ -314,6 +317,9 @@ func (c *JSONBatchEncoder) AppendRowChangedEvent(
 	if err != nil {
 		return errors.Trace(err)
 	}
+	if len(c.terminator) > 0 {
+		value = append(value, []byte(c.terminator)...)
```

Comment:
can we pass `terminator` as `[]byte` to avoid necessary type conversion, it's not free.

Reply:
Done